Sources/ArcGISToolkit/Documentation.docc/Resources/FeatureFormView/FeatureFormViewStep7.swift

```diff
@@ -0,0 +1,129 @@
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct FeatureFormExampleView: View {
+    static func makeMap() -> Map {
+        let portalItem = PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: Item.ID("9f3a674e998f461580006e626611f9ad")!
+        )
+        return Map(item: portalItem)
+    }
+    
+    @StateObject private var dataModel = MapDataModel(
+        map: makeMap()
+    )
+    
+    @State private var identifyScreenPoint: CGPoint?
+    
+    @State private var featureForm: FeatureForm? {
+        didSet { showFeatureForm = featureForm != nil }
+    }
+    
+    @State private var showFeatureForm = false
+    
+    @State private var floatingPanelDetent: FloatingPanelDetent = .full
+    
+    /// A Boolean value indicating whether the alert confirming the user's intent to cancel is displayed.
+    @State private var isCancelConfirmationPresented = false
```

Comment:
Should we call this `cancelConfirmationIsPresented` to fulfill the "reads like an assertion" guideline? I can update the example view separately if you like.